### PR TITLE
Update xspo-alliance-members.json

### PR DESCRIPTION
**Pull request by devopsralf**
D13 Coming in hot!

### CODE REVIEW

**Review by Golden-VS — CHANGES_REQUESTED** [review]
Please remove the "}" in line 75

File: xspo-alliance-members.json [hunk 1]
```diff
@@ -72,7 +72,12 @@
                 "pool_id": "48fd30a022a3daaed7806f5c8739516129b32146f0efd1440c1fa14a",
                 "member_since": "2021-07-20",
                 "name": "NFTS (NFTS Pool)"
-            }            
+            },
+            "10": {
+                "pool_id": "a5db584e578e62611de7f8a4062ad9079a2e59ca33442662d0912a30",
+                "member_since": "2021-07-20",
+                "name": "D13 (DREAM13 Pool)"
+            },
```
Comment (rp5210) [inline xspo-alliance-members.json:80]: please remove the comma ( , ) at the end of this line (80) and submit an updated PR

**Review by Golden-VS — CHANGES_REQUESTED** [review]
Change tag number to 11 (current tag)

**Review by rp5210 — CHANGES_REQUESTED** [review]
See comment on line 80. There is an extra comma after the closing bracket }

**Comment by rp5210** [comment]
> Please remove the "}" in line 75

**Pull request closed without merging**